en/config_vtol/vtol_duo_tailsitter_config.md

```diff
@@ -0,0 +1,94 @@
+:::
+
+As a sanity check, we start off by setting the following gain parameters:
+
+- Roll rate control:
+  - [MC_ROLLRATE_P](../advanced_config/parameter_reference.md#MC_ROLLRATE_P) = [MC_ROLLRATE_I](../advanced_config/parameter_reference.md#MC_ROLLRATE_I) = [MC_ROLLRATE_D](../advanced_config/parameter_reference.md#MC_ROLLRATE_D) = `0`, 
+  - [MC_ROLLRATE_K](../advanced_config/parameter_reference.md#MC_ROLLRATE_K) = `1`
+- Pitch rate control
+  - [MC_PITCHRATE_P](../advanced_config/parameter_reference.md#MC_PITCHRATE_P) = [MC_PITCHRATE_I](../advanced_config/parameter_reference.md#MC_PITCHRATE_I) = [MC_PITCHRATE_D](../advanced_config/parameter_reference.md#MC_PITCHRATE_D) = `0`
+  - [MC_PITCHRATE_K](../advanced_config/parameter_reference.md#MC_PITCHRATE_K) = `1`
+- Yaw rate control
+  - [MC_YAWRATE_P](../advanced_config/parameter_reference.md#MC_YAWRATE_P) = [MC_YAWRATE_I](../advanced_config/parameter_reference.md#MC_YAWRATE_I) = [MC_YAWRATE_D](../advanced_config/parameter_reference.md#MC_YAWRATE_D) = `0`
+  - [MC_YAWRATE_K](../advanced_config/parameter_reference.md#MC_YAWRATE_K) = `1`
+- Angle control
+  - [MC_ROLL_P](../advanced_config/parameter_reference.md#MC_ROLL_P) = [MC_PITCH_P](../advanced_config/parameter_reference.md#MC_PITCH_P) = [MC_YAW_P](../advanced_config/parameter_reference.md#MC_YAW_P) = `0`
+
+The expected behaviour is that when you arm in [Stabilized mode](../flight_modes/manual_stabilized_mc.md), both the motors should spin, control surfaces should be centred, and no effect of manual RC input using the transmistter is observed.
+
+:::note IF you can do so safely!
+Gently pick up the vehicle by hand and try tilting it in different axes.
+You should feel no countering torque from the motors or control surfaces.
+:::
+
+Once this sanity check is complete, we move on to slowly increasing the gains.
+
+### Rate Gains
+
+- Increase [MC_ROLLRATE_P](../advanced_config/parameter_reference.md#MC_ROLLRATE_P) by 0.1.
+  Arm and lift the vehicle and displace it in the roll axis.
+  You should feel a countering torque as the tailsitter tries to reorient itself.
+  This is still a very manual process, and there is not much science to it.
+  Increase the proportional gain until you can feel sustained oscillations when you displace it.
+  Then decrease the gain to 1/2 of that value.
+  
+  If your oscillations never converge, it is worth trying to increase the [MC_ROLLRATE_I](../advanced_config/parameter_reference.md#MC_ROLLRATE_I) gain to a very small value.
+  If oscillations are the problem, try increasing [MC_ROLLRATE_D](../advanced_config/parameter_reference.md#MC_ROLLRATE_D)
+
+- Repeat the above process for Pitch and Yaw.
+
+
+### Angle Gains
+
+Once you have a tuned rate controller, your vehicle can hover in a very stable way (albeit with some support for position control, since your angle gains are still 0 and your RC transmitter will not work).
```

Comment:
By "albeit .." are you just saying "in theory, because you don't have any position control or controller? Why won't your RC transmitter work? I don't get this.

Reply:
Since your attitude controller gains are 0, the vehicle wont act on any external RC inputs. Any angle setpoints you give using your RC transmitter wont affect the attitude of the vehicle.

Reply:
Then I think you mean "no support"?

```suggestion
Once you have a tuned rate controller, your vehicle can hover in a very stable way (albeit with no support for position control, since your angle gains are still 0 and your RC transmitter will not work).
```

Reply:
By that line in the bracket, I mean that you would physically need to try and keep the vehicle hovering in one position, your RC transmitter won't work. "some support" is basically you yourself keeping the vehicle roughly in one place so it doesn't drift about a lot.